source/types.ts

```diff
@@ -137,6 +137,13 @@ export interface Options<T> {
 	*/
 	clearInvalidConfig?: boolean;
 
+	/**
+	The mode that will be used for the config file.
+
+	@default 0o666
+	*/
+	mode: number | string;
```

Comment:
```suggestion
	configFileMode: number;
```

I'm aware the underlying API supports a string, but that's only for legacy reasons. I prefer to keep it simple.